src/version/version_test.go

```diff
@@ -0,0 +1,146 @@
+package version
```

Comment:
what do you think isolate your test file using _test @esequielvirtuoso

Reply:
Done.